packages/comby-kernel/comby-kernel.1.4.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "rvantonder@gmail.com"
+authors: "Rijnard van Tonder"
+homepage: "https://github.com/comby-tools/comby"
+bug-reports: "https://github.com/comby-tools/comby/issues"
+dev-repo: "git+https://github.com/comby-tools/comby.git"
+license: "Apache-2.0"
+build: [
+    [
+      "dune"
+      "build"
+      "-p"
+      name
+      "-j"
+      jobs
+      "@install"
+      "@runtest" {with-test}
+    ]
+]
+depends: [
+  "dune" {>= "2.7.0"}
+  "ocaml" {>= "4.08.1"}
+  "angstrom" {>= "0.15.0"}
+  "core_kernel"
```

Comment:
```suggestion
  "core_kernel"
  "camlzip" {with-test}
```

From ```
#=== ERROR while compiling comby-kernel.1.4.1 =================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/comby-tools/comby/archive/1.4.1.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/comby-kernel.1.4.1
# command              ~/.opam/4.08/bin/dune build -p comby-kernel -j 47 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/comby-kernel-19-e7966f.env
# output-file          ~/.opam/log/comby-kernel-19-e7966f.out
### output ###
# File "test/alpha/dune", line 15, characters 2-9:
# 15 |   camlzip))
#        ^^^^^^^
# Error: Library "camlzip" not found.
# Hint: try:
#   dune external-lib-deps --missing -p comby-kernel -j 47 @install @runtest
# File "test/common/dune", line 70, characters 2-9:
# 70 |   camlzip))
#        ^^^^^^^
# Error: Library "camlzip" not found.
# Hint: try:
#   dune external-lib-deps --missing -p comby-kernel -j 47 @install @runtest
# File "test/omega/dune", line 16, characters 2-9:
# 16 |   camlzip))
#        ^^^^^^^
# Error: Library "camlzip" not found.
# Hint: try:
#   dune external-lib-deps --missing -p comby-kernel -j 47 @install @runtest
```

Reply:
It is still failing, I am not sure why now, since camlzip is clearly installed

Reply:
@rvantonder the issue seems to be in dune to me (I've raised the issue with upstream), however here is a work around to make your tests compile:
```
diff --git a/dune-project b/dune-project
index 79a44da..8402871 100644
--- a/dune-project
+++ b/dune-project
@@ -1,2 +1,2 @@
-(lang dune 2.7)
+(lang dune 2.8)
 (name comby)
diff --git a/test/alpha/dune b/test/alpha/dune
index ef2afcb..b9f9ec5 100644
--- a/test/alpha/dune
+++ b/test/alpha/dune
@@ -1,5 +1,6 @@
 (library
  (name alpha_test_integration)
+ (package comby)
  (modules
   test_custom_metasyntax
   test_optional_holes
diff --git a/test/common/dune b/test/common/dune
index 26ce2d6..4824447 100644
--- a/test/common/dune
+++ b/test/common/dune
@@ -1,5 +1,6 @@
 (library
  (name common_test_integration)
+ (package comby)
  (modules
   test_helpers
   test_extract_regex
diff --git a/test/omega/dune b/test/omega/dune
index 1801d80..3b31a7e 100644
--- a/test/omega/dune
+++ b/test/omega/dune
@@ -1,5 +1,6 @@
 (library
  (name omega_test_integration)
+ (package comby)
  (modules
 ;
 ; TODO
```

Reply:
Got it, thanks, will do this!